### ISSUE TITLE
Add typescript support

### ISSUE THREAD

**Issue by mohitxskull** [cashfree/cashfree-js#1]

**Comment by MrBns** [comment]
Yeah.. there is no type support for this package..  may be it won't possible because they are just serving bundled code ?

**Comment by MrBns** [comment]
> Yeah.. there is no type support for this package.. may be it won't possible because they are just serving bundled code ?

Ohh I see .. its some javascript code not bundled.  so yeah its possible to provide typescript support ..

**Comment by Ayush-pbh** [comment]
TypeScript Support Not present. Please add Typescript support

**Comment by rajnandan1** [comment]
@Harshith-Kanigalpula can we add typescript support

**Comment by Harshith-CF** [comment]
Create a declaration file decs.d.ts file in the root of the folder
declare module '@cashfreepayments/cashfree-js'; 

In tsconfig.json find the include array and append decs.d.ts in it
{ .... "include": [ "src", "decs.d.ts" ] }
